common/util/web_controller/autoware_web_controller/package.xml

```diff
@@ -40,7 +40,7 @@
   <!--   <test_depend>gtest</test_depend> -->
   <!-- Use doc_depend for packages you need only for building documentation: -->
   <!--   <doc_depend>doxygen</doc_depend> -->
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Since `ament_cmake_auto` is used, could we change use that here instead of `ament_cmake`?